tox.ini

```diff
@@ -47,25 +46,26 @@ passenv =
     DATABASE
     CACHE
 commands =
-    python manage.py migrate
-    python manage.py missing_migrations
+    pdm run python manage.py migrate
+    pdm run python manage.py missing_migrations
```

Comment:
What's the reason for the `pdm run`  before all of these? The whole point of tox is to create an isolated environment already, but I'm not familiar with pdm and tox+pdm so please enlighten me.

Reply:
`pdm run` is supposed to be added to the front implicitly, but for some reason it does not ... Found this related issue https://github.com/pdm-project/tox-pdm/issues/12